### PR TITLE
cat: use airbyte-protocol-model~=0.4.0

### PR DESCRIPTION
**Pull request by alafanechere**
CAT was not pinning its dependecy to airbyte-protocols-models. It led to the installation of an old version which is semantically higher than the most recent one.

### CODE REVIEW

File: airbyte-integrations/bases/connector-acceptance-test/pyproject.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ homepage = "https://github.com/airbytehq/airbyte"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-airbyte-protocol-models = "*"
+airbyte-protocol-models = ">=0.4.0,<0.5.0"
```
Comment (maxi297) [inline airbyte-integrations/bases/connector-acceptance-test/pyproject.toml:15]: Should we upper bound to a major even? I guess there would be no breaking changes until a major is changed which mean minor changes are probably fine

**Review by maxi297 — APPROVED** [review]
LGTM!

**Comment by alafanechere** [comment]
/legacy-publish connector=bases/connector-acceptance-test run-tests=false

**Comment by alafanechere** [comment]
/legacy-publish connector=bases/connector-acceptance-test run-tests=false

**Comment by alafanechere** [comment]
/legacy-publish connector=bases/connector-acceptance-test run-tests=false

**Comment by alafanechere** [comment]
/legacy-publish connector=bases/connector-acceptance-test run-tests=false

**Comment by alafanechere** [comment]
Publishing manually in https://github.com/airbytehq/airbyte/actions/runs/5895733022 as the slash command is down.

**Pull request merged**